### PR TITLE
Dcwither/update node to lts

### PR DESCRIPTION
**Pull request by dcwither**
### Summary:
This should fix issues with building on node version 14.x.  I've also updated the workflows to run on 10,12,14 instead of 8,10,12.

### Test Plan:
Local:

- install node 14.5.0
- reinstall dependencies
- `npm start`

CI: run it

### CODE REVIEW

**Comment by lesliecodes** [comment]
Hm.. I just tried this out but it fails 😢

**Comment by dcwither** [comment]
@lesliecodes can you share the failure? Was it the arg types one, or a node-sass one? For node sass you need to uninstall all the dependencies again.

File: .github/workflows/size-limit.yml [hunk 1]
```diff
@@ -6,6 +6,11 @@ on:
 jobs:
   size:
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```
Comment (anniehu4) [inline .github/workflows/size-limit.yml:12]: why is this one running on 12 instead of 14?

**Review by anniehu4 — COMMENTED** [review]
is the bundle size failure expected still?

**Comment by lesliecodes** [comment]
It works! I did have to remove all the old /node_modules

File: .github/workflows/size-limit.yml (hunk 1 above)
Comment (dcwither) [reply .github/workflows/size-limit.yml:12]: Didn't want to mess with that one yet, was hoping it would prevent it from failing. I don't think it has.

Comment (dcwither) [reply .github/workflows/size-limit.yml:12]: Once this is merged, if the next one fails, I'll look into it more.

**Review by anniehu4 — APPROVED** [review]

**Pull request merged**